packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -0,0 +1,444 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:developer' as developer;
+import 'dart:math' as math;
+import 'dart:ui' as ui show window;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/material.dart';
+import 'package:vector_math/vector_math_64.dart';
+
+import 'basic.dart';
+import 'binding.dart';
+import 'framework.dart';
+import 'gesture_detector.dart';
+
+/// A widget that enables inspecting the child widget's structure.
+///
+/// This widget is useful for understand how an app is structured enabling
+/// interactive
```

Comment:
nit: docs should have full grammar and punctuation, so there's a missing period here.